docs/dev/news/explicit-failing-assertions.feat

```diff
@@ -0,0 +1,13 @@
+Code actions in editor to explicit failing assertions.
+In VSCode, place the cursor on a failing assertion that support being made explicit and either
+- Position the caret on a failing assertion, press CTRL+; and then ENTER
+- Hover over the failing division by zero, click "quick fix", press ENTER
```

Comment:
Out of scope for this PR, but I think we should mention this feature in the readme for https://github.com/dafny-lang/ide-vscode too! And in the details of the extension (if that isn't automatically the same content)